sheetfu/modules/table.py

```diff
@@ -130,6 +150,27 @@ def commit(self):
         self.batches = list()
         return response
 
+    def select(self, filters):
+        """
+        Method to filter items on a table based on field name and value.
+        Values on the same header will be filtered using OR
+        Values on different headers will be filtered using AND
+
+        :param filters: Dictionary that has header as key and an List (or a single item) of values
+        as the value. Example: {"Version": ["1.2", "1.4", "1.3"], "body": ["test"]}
```

Comment:
In sheetfu apps script we use CNF as a specification for the select method. I'm not sure if it's the best one, but I think it would be good to keep them consistent between each other.

An array used as filter as an AND of ORs (see CNF). Examples:
 * >>> [{date: today}, [{tag: 1},{tag: 2}]] // (date === today && (tags === 1 || tags === 2))
 * >>> [[{assigneeId: 'GO'}, {assigneeId: 'AM'}]] // (assigneeId === 'GO' || assigneeId === 'AM')
 * >>> [{name: 'Guillem'}, {surname: 'Orpinell'}] // (name === 'Guillem' && surname === 'Orpinell')
 * >>> {name: 'Guillem', surname: 'Orpinell'} // (name === 'Guillem' && surname === 'Orpinell')

The code could actually be just translated from JS to python from the GAS version of sheetfu:
https://github.com/socialpoint-labs/sheetfu-apps-script/blob/master/selector.gs